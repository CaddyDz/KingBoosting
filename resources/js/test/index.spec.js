import {
    mount
} from '@vue/test-utils'
import ExampleComponent from '../components/AppBarComponent.vue'

test('it works', () => {
    expect(1 + 1).toBe(2)
})

test('should mount without crashing', () => {
    const wrapper = mount(ExampleComponent)

    expect(wrapper).toMatchSnapshot()
})
