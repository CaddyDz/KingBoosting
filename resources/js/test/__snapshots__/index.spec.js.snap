// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should mount without crashing 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
